autosubmit/git/autosubmit_git.py

```diff
@@ -260,6 +261,19 @@ def clone_repository(as_conf, force, hpcarch):
 
         return True
 
+    @staticmethod
+    def is_github_repo(git_repo:str) -> bool:
+        git_repo = git_repo.lower().strip()
+
+        git_url_pattern = re.compile(
+            r'^(?:git|ssh|https?|git@[\w\.]+):(//)?[\w\.@\:/\-~]+\.git/?$'
```

Comment:
The `git@` didn't work for me.

https://regex101.com/r/nP09LO/1

The cloned repository is a repo that Luiggi and I used for EDITO, from MOi's GitLab.

Reply:
Thanks Irene! Will test & review it tomorrow :muscle: